ml-agents/mlagents/trainers/behavior_id_utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, NamedTuple
+
+
+class BehaviorIdentifiers(NamedTuple):
+    brain_name: str
+    behavior_ids: Dict[str, int]
+
+    @staticmethod
+    def from_name_behavior_id(name_behavior_id: str) -> "BehaviorIdentifiers":
+        """
+        Parses a name_behavior_id into a BehaviorIdentifiers NamedTuple.
+        This allows you to access the brain name and distinguishing identifiers
+        without parsing more than once.
+        :param name_behavior_id: String of behavior params in HTTP format.
```

Comment:
The format is the "query string".  Maybe it would be good to give an example of what this is parsing?  It wasn't clear to me and I'm familiar with HTTP query strings.

Even just `behavior_ids` -- It wasn't immediately clear which what the `str` is versus the `int` value.  I would expect a member called `behavior_ids` to be a list.